### PR TITLE
[Fix] Address build error and warnings in ConfigureBasemapStyleParametersView.swift

### PR DESCRIPTION
**Pull request by philium**
Related to recent API changes. They weren't intended to introduce build errors, so these changes and the API changes may need to be reverted before the next release. For now, addressing them was the easiest approach.

### CODE REVIEW

**Review by zkline101 — APPROVED** [review]

**Review by yo1995 — COMMENTED** [review]
Overall looks good

File: Shared/Samples/Configure basemap style parameters/ConfigureBasemapStyleParametersView.swift [hunk 1]
```diff
@@ -20,70 +20,56 @@ struct ConfigureBasemapStyleParametersView: View {
...
-                        // Use the default language setting for the basemap style.
-                        BasemapStyleLanguage.default,
-                        // Use the global language (English) for basemap labels.
-                        BasemapStyleLanguage.global,
-                        // Uses country-local language for basemap labels.
-                        BasemapStyleLanguage.local,
-                        // Use the system locale language for basemap labels.
-                        BasemapStyleLanguage.applicationLocale
-                    ],
+                    [BasemapStyleLanguageStrategy.default, .global, .local, .applicationLocale],
```
Comment (yo1995) [inline Shared/Samples/Configure basemap style parameters/ConfigureBasemapStyleParametersView.swift:40]: Can you retain the comments so the audience can see the difference between the strategies more easily? Thank you.

Comment (philium) [reply Shared/Samples/Configure basemap style parameters/ConfigureBasemapStyleParametersView.swift:40]: Aren't the comments simply repeating what is in the documentation?

Comment (yo1995) [reply Shared/Samples/Configure basemap style parameters/ConfigureBasemapStyleParametersView.swift:40]: Yes, I thought adding some comment would help the reader to spot the key place to look at in the code. OK not to add.

**Review by yo1995 — APPROVED** [review]

**Pull request merged**